dev/io.openliberty.session.6.1.internal/bnd.bnd

```diff
@@ -0,0 +1,46 @@
+#*******************************************************************************
+# Copyright (c) 2024 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: WAS Session
+Bundle-SymbolicName: io.openliberty.session.6.1.internal
+Bundle-Description: WAS Session for Servlet 6.1
```

Comment:
Should WAS be changed to Liberty?

Reply:
WAS is being used in other bundles (webcontainer.servlet.6.1.internal and servlet.6.1.internal.factories).  If it should be changed to Liberty Session, WAS WebContainer should also be changed to Liberty WebContainer in other bundles.

Reply:
Some of the Web Container code could be common with traditional WebSphere, but the EE 11 code will not be common with traditional WebSphere since there is no support of Java EE past EE 7.  Since it is a new bundle I figured it was worth asking.  I am not saying to change it everywhere, but here might be a good place since you are making it anew.

Reply:
I would have changed the Bundle-Name as well, but I can live with it.